### PR TITLE
Fixes #207 - Add synthetic history records for keywords and fix a race condition

### LINKED ISSUE #207: Add synthetic history records for keywords and fix a race condition when some history records are not saved
We've noticed some entries are not recorded if a bug was edited during the time when the import is running.
In addition, bugs filed with keywords already set, don't have records about that in bugzilla, so we should create synthetic ones.

### PR DESCRIPTION
**Pull request by ksy36**
Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is
  referenced, the pull request should include the bug number in the title)
- [ ] Scan the PR and verify that no changes (particularly to
  `.circleci/config.yml`) will cause environment variables (particularly
  credentials) to be exposed in test logs
- [ ] Ensure the container image will be using permissions granted to
  [telemetry-airflow](https://github.com/mozilla/telemetry-airflow/)
  responsibly.

### CODE REVIEW

**Review by jgraham — APPROVED** [review]

**Comment by ksy36** [comment]
@scholtzan I've moved changes from the other [PR](https://github.com/mozilla/docker-etl/pull/208) to this one. It comes from a branch instead of a fork, so CI did run, but it still needs a review from someone with write access. 
Could you please r+ this one as well?

**Review by scholtzan — APPROVED** [review]

**Comment by scholtzan** [comment]
I added jgraham to have write access

**Comment by ksy36** [comment]
thank you :)

**Pull request merged**
